src/go/k8s/controllers/redpanda/console_controller_test.go

```diff
@@ -342,4 +342,49 @@ var _ = Describe("Console controller", func() {
 			}, timeout, interval).Should(BeTrue())
 		})
 	})
+
+	Context("When enabling multiple Login providers", func() {
+		ctx := context.Background()
+		It("Should prioritize RedpandaCloud", func() {
+			var (
+				rpCloudDomain   = "test.auth.vectorized.io"
+				rpCloudAudience = "dev.vectorized.io"
+			)
+
+			By("Updating Console RedpandaCloud Login fields")
+			console := &redpandav1alpha1.Console{}
+			Expect(k8sClient.Get(ctx, types.NamespacedName{Namespace: ConsoleNamespace, Name: ConsoleName}, console)).Should(Succeed())
+			console.Spec.Login.RedpandaCloud = &redpandav1alpha1.EnterpriseLoginRedpandaCloud{
+				Enabled:  true,
+				Domain:   rpCloudDomain,
+				Audience: rpCloudAudience,
+			}
+			Expect(k8sClient.Update(ctx, console)).Should(Succeed())
+
+			By("Having only RedpandaCloud provider in ConfigMap")
```

Comment:
This test doesn't check if other provider `google` is configured.

Reply:
The test for configmap is in other context [REF](https://github.com/redpanda-data/redpanda/blob/dev/src/go/k8s/controllers/redpanda/console_controller_test.go#L210) The naming is a bit weird though as it calls it "enterprise features", Google was the first one developed for SSO

Reply:
I saw that test, but the problem I see if you configure both SSO providers in console CR you should not see anything in configmap that relates to google SSO, right? That's why I was thinking the test should check if RedpandaCloud is set and google is not.

Reply:
Oh, I think I have it on this part [REF](https://github.com/redpanda-data/redpanda/pull/6359/files#diff-0a3cbd8733e6260bf1fb67f02a224a7a659564df9526cdc7333df6a2d655db7cR378)

Reply:
Sorry 🙈 you are totally right THANK YOU!